lms/djangoapps/discussion/plugins.py

```diff
@@ -1,33 +1,25 @@
 """
 Views handling read (GET) requests for the Discussion tab and inline discussions.
 """
+from django.utils.translation import ugettext_lazy as _
 
+from openedx.core.djangoapps.discussions.discussions_apps import DiscussionApp
+from .django_comment_client.utils import is_discussion_enabled
 
-from django.conf import settings
-from django.utils.translation import ugettext_noop
 
-import lms.djangoapps.discussion.django_comment_client.utils as utils
-from lms.djangoapps.courseware.tabs import EnrolledTab
-from xmodule.tabs import TabFragmentViewMixin
-
-
-class DiscussionTab(TabFragmentViewMixin, EnrolledTab):
```

Comment:
Maybe I'm missing something - don't we need to keep this old code around so that the existing/legacy discussions integration still works?

Reply:
The existing discussions will continue working via the plugin route. 
i.e. this same code is now directed through a plugin instead of working directly.